homework/index.js

```diff
@@ -30,13 +30,175 @@
     return elem;
   }
 
+  function createHtml(jsonData) {
```

Comment:
Overall, I suggest creating more specific-purpose functions to create parts of your application.